packages/devtools_server/lib/src/handlers.dart

```diff
@@ -83,18 +87,88 @@ class ServerApi {
     switch (request.url.path) {
       case 'api/logScreenView':
         return api.logScreenView(request);
+
+      // ----- Flutter Tool GA store. -----
+      case 'api/getFlutterGAEnabled':
+        // Is Analytics collection enabled?
+        return api.getCompleted(
+          request,
+          '${Usage.doesStoreExits ? _usage.enabled : 'null'}',
+        );
+      case 'api/getFlutterGAClientId':
+        // Flutter Tool GA clientId - ONLY get Flutter's clientId if enabled is
+        // true.
+        return (Usage.doesStoreExits)
+            ? api.getCompleted(
+                request, _usage.enabled ? _usage.clientId : 'null')
+            : api.getCompleted(request, 'null');
+
+      // ----- DevTools GA store. -----
+      case 'api/resetDevTools':
+        _devToolsUsage.reset();
+        return api.getCompleted(request, 'true');
+      case 'api/getDevToolsFirstRun':
+        // Has DevTools been run first time? To bring up welcome screen.
+        return api.getCompleted(request, '${_devToolsUsage.isFirstRun}');
+      case 'api/getDevToolsEnabled':
+        // Is DevTools Analytics collection enabled?
+        return api.getCompleted(request, '${_devToolsUsage.enabled}');
+      case 'api/setDevToolsEnabled':
+        // Enable or disable DevTools analytics collection.
+        final queryParams = request.requestedUri.queryParameters;
+        _devToolsUsage.enabled = queryParams.containsKey('enabled')
+            ? queryParams['enabled'] == 'true'
+            : false;
+        return api.setCompleted(request, '${_devToolsUsage.enabled}');
+
+      // ----- DevTools survey store. -----
+      case 'api/getSurveyActionTaken':
+        // SurveyActionTaken has the survey be taken?
```

Comment:
r/survey be taken/survey been acted upon (taken or dismissed).

Reply:
done.